packages/teleport/src/cluster/components/Audit/AuditEvents/__snapshots__/AuditEvents.story.test.tsx.snap

```diff
@@ -1396,7 +1528,7 @@ exports[`rendering of Events 1`] = `
           <td
             style="word-break: break-word;"
           >
-            User [admin@example.com] has ended non-interactive session [9febab45-6491-11e9-80a1-427cfde50f5a] on node [undefined] 
+            User [admin@example.com] has ended an non-interactive session [9febab45-6491-11e9-80a1-427cfde50f5a] on node [undefined] 
```

Comment:
Not sure if this should be `a non-interactive` too?

Reply:
thank you, i forgot to run my tests before commit :)